spec/tasks/send_survey_notifications_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+describe 'send_survey_notifications rake task' do
+  let!(:intake) { create :state_file_az_intake, email_address: "test@example.com", email_address_verified_at: 1.minute.ago }
+  let!(:efile_submission) { create :efile_submission, :for_state, :accepted, data_source: intake }
+  let!(:efile_submission_no_intake) { create :efile_submission, :for_state, :accepted, data_source: nil }
+  before(:all) do
+    Rails.application.load_tasks
+  end
+  it 'runs without error even when submission does not have an intake' do
```

Comment:
Well the efile submission table has intakes from 2021 in production.  They must be from CTC / GYR since no they do not have state filing intakes.  I left some comments on the ticket yesterday with a more thorough explanation.

Reply:
Ooo so the `efile_submission_no_intake` is supposed to represent a ctc efile_submission? @rickreyhsig

Reply:
if thats the case you should remove the `for_state` trait as that is adding a statefile data source and just do something like this `let!(:ctc_efile_submission) { create :efile_submission, :accepted}`

Reply:
Right, I wrote the specs before having a full picture of what was wrong.